grunttasks/jscs.js

```diff
@@ -14,7 +14,8 @@ module.exports = function (grunt) {
       src: [
         '<%= yeoman.app %>/**/*.js',
         '!<%= yeoman.app %>/bower_components/**',
-        '!<%= yeoman.app %>/scripts/vendor/**'
+        '!<%= yeoman.app %>/scripts/vendor/**',
+        '!<%= yeoman.app %>/scripts/lib/bloom*.js'
```

Comment:
what are the jscs conflicts in the bloomfilter file? also see https://github.com/mozilla/fxa-content-server/pull/2810/files#r35326816